aptos-move/framework/move-stdlib/sources/string.move

```diff
@@ -87,7 +87,7 @@ module std::string {
 
 
     // Native API
-    native fun internal_check_utf8(v: &vector<u8>): bool;
+    public native fun internal_check_utf8(v: &vector<u8>): bool;
     native fun internal_is_char_boundary(v: &vector<u8>, i: u64): bool;
```

Comment:
Should we just make all these native fun publics?

Reply:
I wouldn't do for now. Turning something from private to public is not a breaking change, but the opposite would be. The others are mostly for implementing standard string functions here, and not of much use elsewhere I guess.